.github/workflows/examples.yml

```diff
@@ -5,9 +5,6 @@ on:
     branches:
       - master
   pull_request:
-    paths:
-      - gnovm/**/*.gno
```

Comment:
Why would we not run them for just these changes?

What does a change in any other part of the codebase have to do with the examples? I'd like to see real examples of this issue

Reply:
#2491 was green in the PR, but was [red after being merged](https://github.com/gnolang/gno/commit/36cf380542f48540eb3b74c57006f4403157628f), because [a txtar test was no longer correct](https://github.com/gnolang/gno/pull/2582).